src/pages/verify/integrations/javascript.mdx

```diff
@@ -0,0 +1,292 @@
+- `prompt (string)`: The type of user interaction that is required. `prompt: 'login'` will ignore an existing session and force a new login. `prompt: 'none'` will check if the user has an existing session: Criipto Verify will never require any user interactions in this case. The default value is `login`, which will always force the user to re-authenticate. Read more [here](./guides/sso/#per-request-options).
+
+```js
+      redirectUri: redirectUri!,
+      responseMode: params.responseMode || 'query',
+      responseType: responseType!,
+      acrValues: acrValues,
+      pkce: params.pkce,
+      state: params.state,
+      loginHint: params.loginHint,
+      uiLocales: params.uiLocales,
+      extraUrlParams: params.extraUrlParams,
+      scope: scope,
+      prompt: params.prompt,
+      nonce: params.nonce
+```
+
+Authorization parameters can also be provided in the `CriiptoAuth` constructor, as shown in the example below.
+
+```js
+var criiptoAuth = new CriiptoAuth({
+  domain: '{YOUR_CRIIPTO_DOMAIN}',
+  clientID: '{YOUR_CRIIPTO_APPLICATION_ID}',
+  store: sessionStorage,
+  redirectUri: 'http://localhost:8000/example/index.html',
+  acrValues: 'urn:grn:authn:dk:nemid:poces',
+});
+```
+
+## Authorization methods
+
+You can use the following methods to authorize the user:
+
+- [`criiptoAuth.popup.authorize`](#authorization-with-popup) - Opens a popup window to authenticate the user.
+- [`criiptoAuth.redirect.authorize`](#authorization-with-redirect) - Redirects the user to the Criipto Verify login page.
+- [`criiptoAuth.authorizeResponsive`](#authorization-with-popup-or-redirect-based-on-the-screen-size) - Opens a popup window if the user is on a mobile device, otherwise redirects the user to the Criipto Verify login page.
+
+### Authorization with popup
+
+Will open a popup window to authenticate the user.
+
+#### authorize
+
+```html
+<!-- index.html -->
+...
+<button id="login-button-popup">Login</button>
+<div id="result"></div>
+...
+
+<script type="text/javascript">
+  document
+    .getElementById('login-button-popup')
+    .addEventListener('click', () => {
+      criiptoAuth.popup
+        .authorize({
+          width: 300,
+          height: 400,
+          redirectUri: 'http://localhost:8000/example/index.html',
+          acrValues: 'urn:grn:authn:dk:nemid:poces',
+        })
+        .then((result) => {
+          console.log(result, result.id_token, result.claims);
+          document.getElementById('result').textContent =
+            JSON.stringify(result);
+        });
+    });
+</script>
+```
+
+- `width (optional, number)`: Popup width
+- `height (optional, number)`: Popup height
+- For `redirectUri` and `acrValues`, see [authorization parameters](#authorization-parameters)
```

Comment:
Can remove these, and perhaps just link directly to the full set of parameters in the github documentation.